www/src/pages/index.mdx

```diff
@@ -41,38 +41,21 @@ import { ComponentName } from '@edx/paragon';
 
 #### SCSS Foundation
 
-Usage for Open edX and others:
+Usage with no theme:
 
 ```scss
-// ... Any custom SCSS variables should be defined here
-@import "~@edx/paragon/scss/core/core.scss';
+@import "@edx/paragon/scss/core/core.scss";
 ```
 
-Usage on edx.org:
+Usage with a theme:
 
 ```scss
-@import "~@edx/paragon/scss/edx/theme.scss';
-@import '~@edx/paragon/scss/edx/fonts.scss'; // Roboto
+@import "@my-brand/fonts.scss";
+@import "@my-brand/variables.scss";
```

Comment:
Is it worth talking about the difference between SCSS variables - and how first-one-in-wins - vs the overrides and how they go below the core?  A sentence or two might go a long way to improving and solidifying peoples' mental models.  I know I personally find a it a bit confusing.